lib/maxval.js

```diff
@@ -3,9 +3,9 @@ function Maxval() {
 }
 module.exports = Maxval;
 
-var Error = require(__dirname+"/error.js");
+var Error = require("./error.js");
 
 Maxval.prototype.toJSON = function() {
-  throw new Error.ReqlRuntimeError('Cannot convert `r.maxval` to JSON', query.frames)
+  throw new Error.ReqlRuntimeError('Cannot convert `r.maxval` to JSON' /* ?? , query.frames */)
```

Comment:
This is a bit tricky to fix, we can just remove the query for now.
To properly fix it, we need to add a new internal option to determine if `r.asc/desc` is valid in the given context.